internal/bundler/bundler.go

```diff
@@ -93,15 +94,15 @@ func parseFile(
 	source := logging.Source{
 		Index:        sourceIndex,
 		IsStdin:      isStdin,
-		AbsolutePath: path,
+		AbsolutePath: sourcePath,
 		PrettyPath:   prettyPath,
 		Contents:     contents,
 	}
 
 	// Get the file extension
 	extension := ""
-	if lastDot := strings.LastIndexByte(path, '.'); lastDot >= 0 {
-		extension = path[lastDot:]
+	if lastDot := strings.LastIndexByte(sourcePath, '.'); lastDot >= 0 {
```

Comment:
this line is probably could be simplified with https://golang.org/pkg/path/#Ext thinking @evanw any reason we don't just use it?

Reply:
Sounds good to me. I probably just wasn't aware of it existing.